modules/test/playwright/pages/product-navigation-control-menu-web/ProductMenuPage.ts

```diff
@@ -99,9 +99,19 @@ export class ProductMenuPage {
 		await this.membershipsButton.click();
 	}
 
-	async goToPages() {
-		await this.siteBuilderButton.click();
-		await this.pagesButton.click();
+	async goToPages(siteUrl?: string) {
+
+		/**
+		 * Hard reload work around for
+		 * "Uncaught DOMException: Failed to execute 'removeChild'
+		 * on 'Node': The node to be removed is not a child of this
+		 * node." on Page Editor blocking playwright JS from running.
+		 * frontend-js-react-web/src/main/resources/META-INF/resources/js/render.tsx
+		 */
```

Comment:
I don't think we need the comment here. Using a URL redirect instead of clicking through menus is preferable since it lets us test what is most important instead of just navigation.